modules/passkey/docs/v0.2.0/audit.md

```diff
@@ -0,0 +1,22 @@
+# Audit Results
+
+## Auditor
+
+Certora (<https://www.certora.com/>).
+
+## Notes
+
+The final audit was performed on commit [8a906605010520bed5b532c9d2feb04fdf237832](https://github.com/safe-global/safe-modules/tree/8a906605010520bed5b532c9d2feb04fdf237832).
+
+There is one acknowledged finding from the audit report:
+
+- _I-01: EVM Version Shanghai may not work on other chains due to PUSH0_
+  > We explicitly set the EVM version to target by the Solidity compiler to `paris` to avoid emitting `PUSH0` opcodes and increase compatibility across L2s.
+
+The vendored FreshCryptoLib library implementing the secp256r1 Solidity based verification was not included in this audit. However, the contracts are used in other [audited](https://github.com/base-org/webauthn-sol/tree/main/audits) smart wallets such as the Coinbase Smart Wallet's [`webauthn-sol`](https://github.com/base-org/webauthn-sol) implementation. The [Daimo P-256 verifier]() has been previously [audited](https://github.com/daimo-eth/daimo/tree/master/audits) and is compatible with the Safe WebAuthn signer contracts.
```

Comment:
I would add a permalink to the audits folder to avoid dead links due to restructurings of a 3rd party repository. Alternatively, I would include the reports in our repo.

Reply:
Should be fixed now.